frontend/src/js/previous-queries/list/ProjectItemDragContainer.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC, useRef } from "react";
+import { useDrag } from "react-dnd";
+
+import { getWidthAndHeight } from "../../app/DndProvider";
+import { DNDType } from "../../common/constants/dndTypes";
+import type { DragItemFormConfig } from "../../external-forms/types";
+import type { DragItemQuery } from "../../standard-query-editor/types";
+
+import ProjectItem, { ProjectItemT } from "./ProjectItem";
+import { isFormConfig } from "./helpers";
+import { PreviousQueryT } from "./reducer";
+
+interface PropsT {
+  item: ProjectItemT;
+  onIndicateDeletion: () => void;
+  onIndicateShare: () => void;
+  onIndicateEditFolders: () => void;
+}
+
+const getDragType = (item: PreviousQueryT) => {
+  return item.queryType === "CONCEPT_QUERY"
```

Comment:
Das ist glaube ich etwas gefährlich wenn wir Refactorings machen (Verischertenupload kann sein, dass er umgebaut wird), und es gibt theoretisch noch andere Typen die aber momentan nur über API-Nutzung (und nicht das Frontend) entstehen können. Wobei, wenn wir die Patientenhistorie angehen hier noch TableExport hinkommen kann.

Denke dass hier ein explizites handling sinnvoller wäre.

Reply:
That code was only moved, if you have a suggestion here, could you create a new issue?